include/knowhere/config.h

```diff
@@ -66,6 +66,7 @@ enum PARAM_TYPE {
     DESERIALIZE_FROM_FILE = 1 << 5,
     ITERATOR = 1 << 6,
     CLUSTER = 1 << 7,
+    STATIC = 1 << 8,
```

Comment:
Should we have one type for each static method?

Reply:
NO. The STATIC PARAM_TYPE is added since only a small part of parameters is valid within the static function, so only these parameters should be loaded. Morever, exist param with specific PARAM_TYPE is not fit for the static case (e.g. the hasRawData can be influenced not only by build params, but also load params), so a new PARAM_TYPE is needed